packages/upload-api/src/types/usage.ts

```diff
@@ -13,4 +15,10 @@ export interface UsageStorage {
     space: SpaceDID,
     period: { from: Date; to: Date }
   ) => Promise<Result<UsageData, Failure>>
+  record: (
+    customer: AccountDID,
+    resource: UnknownLink,
```

Comment:
Do we have enough information to identify the blob(s)/shard(s) the resource was served from? Feels like a useful thing to track here, especially if the CID being used is not a root CID, but a sub-DAG. I'm thinking about users querying why a certain CID has used so much egress but it is not one of the root CIDs from `upload/add` - how do we prove it belongs to them?